stellar-multitenant/pkg/tenant/manager.go

```diff
@@ -246,7 +246,7 @@ func (m *Manager) DeactivateTenantDistributionAccount(ctx context.Context, tenan
 			distribution_account_status = 'PENDING_USER_ACTIVATION'
 		WHERE id = $1
 		AND distribution_account_type = 'DISTRIBUTION_ACCOUNT.CIRCLE.DB_VAULT'
-		AND status = 'TENANT_ACTIVATED'
+		AND status != 'TENANT_DEACTIVATED'
```

Comment:
Can you explain why we need to filter by the tenant status? Can't we completely remove this filter?

Reply:
I suppose we don't since we'll never be deactivating an account for a deactivated tenant. I just wanted to include this since it looks semantically correct to me, but I can remove it if we have a preference.